### ISSUE TITLE
Bug (maybe not?). User status is set to "Moderated" after email change

### ISSUE THREAD

**Issue by Yardanico** [nim-lang/nimforum#126]
If users changes his email and confirms it, his status on the forum will change to "Moderated". I don't think this is an intended behaviour.

**Comment by dom96** [comment]
Yeah, this is a bug. I should have introduced a new rank called ``EmailChanged`` in the first place as you suggested.
